yiisoft/yii2-bootstrap/CVE-2015-3397.yaml

```diff
@@ -0,0 +1,8 @@
+title:     JSON Data encoded for use in HTML was not safe to use in IE6/IE7, possible XSS attacks
+link:      http://www.yiiframework.com/news/86/yii-2-0-4-is-released/
+cve:       CVE-2015-3397
+branches:
+    master:
```

Comment:
@cebe Are ALL versions smaller than 2.0.4 affected? Or only the versions [>=2.0.0, <2.0.4]?